### PR TITLE
Restructure Inforz

### PR DESCRIPTION
**Pull request by Rdeisenroth**
- [x] Formatter Configuration
- [x] Improved Directory structure
- [x] Combine packages into one package
- [x] Usage of Graphics Path
- [x] CI

### CODE REVIEW

**Comment by miterion** [comment]
Thanks for the pull request. A few notes before I would merge it:

- [ ] Please remove the VSC Config or put it into a contrib folder

Did you also verify everything still builds correctly? A quick check failed for me, but I did not investigate this further.

**Review by miterion — CHANGES_REQUESTED** [review]
See previous comment

**Comment by Rdeisenroth** [comment]
I had indeed some leftover cached files in my working directory when building, should now be fixed. Additionally, i added a ci to automatically check and build the inforz.

File: .vscode [hunk 1]
```diff
@@ -0,0 +1 @@
+contrib/.vscode/
```
Comment (miterion) [inline .vscode]: Please remove this symlink, people can add it manually if they desire so

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Inforz - Informatikerzeitschrift der Fachschaft Informatik an der TU Darmstadt
 
+[![Build LaTeX document](https://github.com/Rdeisenroth/inforz/actions/workflows/build.yml/badge.svg)](https://github.com/Rdeisenroth/inforz/actions/workflows/build.yml)
```
Comment (miterion) [inline README.md]: Can you point this to the d120 inforz repo instead?

**Review by miterion — CHANGES_REQUESTED** [review]
Thanks a lot for the changes. The CI is a good idea and should be quite helpful. Just two little things and I think we can merge.

File: .vscode (hunk 1 above)
Comment (Rdeisenroth) [reply .vscode]: Fixed in [d732d4e](https://github.com/d120/inforz/pull/1/commits/d732d4e4e2436c7ec434092d098348391f17e0f3)

File: README.md (hunk 2 above)
Comment (Rdeisenroth) [reply README.md]: Fixed in [001d375](https://github.com/d120/inforz/pull/1/commits/001d375e02fe932d8ac57228533fc1060bfb6f3e)

**Comment by miterion** [comment]
Thanks a lot!

**Pull request merged**